src/SDKs/ProvisioningServices/ProvisioningServices.Tests/ProvisioningServices.Tests.csproj

```diff
@@ -0,0 +1,45 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+  <Import Project="$([MSBuild]::GetPathOfFileAbove('AzSdk.test.reference.props'))" />
+  <PropertyGroup>
+    <PackageId>ProvisioningServices.Tests</PackageId>
+    <Description>ProvisioningServices.Tests Class Library</Description>
+    <AssemblyName>ProvisioningServices.Tests</AssemblyName>
+    <VersionPrefix>1.0.1-preview</VersionPrefix>    
+  </PropertyGroup>
+
+  <PropertyGroup>
+    <TargetFrameworks>netcoreapp1.1</TargetFrameworks>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Microsoft.Azure.Management.EventHub" Version="2.0.2" />
+    <PackageReference Include="Microsoft.Azure.Management.ServiceBus" Version="0.2.0-preview" />
+  </ItemGroup>
+
+  <ItemGroup>
+    <!--<PackageReference Include="Microsoft.Azure.Management.ProvisioningServices" Version="1.1.0" />-->
+
+    <ProjectReference Include="..\..\IotHub\Management.IotHub\Microsoft.Azure.Management.IotHub.csproj" />
+
+    <ProjectReference Include="..\Management.ProvisioningServices\Microsoft.Azure.Management.ProvisioningServices.csproj" />
+  </ItemGroup>
+  
+  <ItemGroup>
+    <None Update="SessionRecords\**\*.json">
+      <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
+    </None>
+  </ItemGroup>
+
+  <!--Do not remove until VS Test Tools fixes #472-->
+  <ItemGroup>
+    <Service Include="{82a7f48d-3b50-4b1e-b82e-3ada8210c358}" />
+  </ItemGroup>
+  <ItemGroup>
```

Comment:
@montgomp remove line 37 to 44.
all the references are being added to every test project